draft-irtf-cfrg-hpke.md

```diff
@@ -984,6 +1057,19 @@ For X25519 and X448, public keys and Diffie-Hellman outputs MUST be validated
 as described in {{RFC7748}}. In particular, recipients MUST check whether
 the Diffie-Hellman shared secret is the all-zero value and abort if so.
 
+### Future KEMs {#future-kems}
+
+{{kem-security}} lists security requirements on a KEM used within HPKE.
+
+The `AuthEncap()` and `AuthDecap()` functions are OPTIONAL. If a KEM algorithm
+does not provide them, only the Base and PSK modes of HPKE are supported.
+Future specifications which define new KEMs MUST indicate whether or not
+Auth and AuthPSK modes are supported.
```

Comment:
Should we add a column to the registry to reflect this?

Reply:
I think this could be a good idea. I drafted one with the most recent commit.